### PR TITLE
feat: add data sorting to TopQueries table

### LINKED ISSUE #996: Add data sorting to `TopQueries` table
Descending sort by `CPUTimeUs` should be added to table

### PR DESCRIPTION
**Pull request by astandrik**
closes #996

### CODE REVIEW

File: src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx [hunk 1]
```diff
@@ -97,8 +98,8 @@ const queryHashColumn: Column<KeyValueRow> = {
 
 const durationColumn: Column<KeyValueRow> = {
     name: TOP_QUERIES_COLUMNS_IDS.Duration,
-    header: 'Duration, ms',
-    render: ({row}) => formatNumber(parseUsToMs(row.Duration ?? undefined)),
+    header: 'Duration',
+    render: ({row}) => `${formatNumber(parseUsToMs(row.Duration ?? undefined))} ms`,
```
Comment (Raubzeug) [inline src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx:102]: lets add `ms` via i18n.

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx (hunk 1 above)
Comment (astandrik) [reply src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx]: fixed

**Comment by artemmufazalov** [comment]
There will be a link to this PR near commit in changelog, issue id in the name looks redundant

**Review by Raubzeug — APPROVED** [review]
Please fix PR name as Artem mentioned!

**Pull request merged**